lib/Sema/CSSyntacticElement.cpp

```diff
@@ -1738,9 +1738,10 @@ class SyntacticElementSolutionApplication
       }
     }
 
-    TypeChecker::checkSwitchExhaustiveness(
-        switchStmt, context.getAsDeclContext(), limitExhaustivityChecks);
-
+    if (!hadError) {
+      TypeChecker::checkSwitchExhaustiveness(
+          switchStmt, context.getAsDeclContext(), limitExhaustivityChecks);
```

Comment:
I would just check exhaustiveness anyways but throw the limited mode switch so folks don’t have to build twice to see _something_.

Reply:
Makes sense, thanks!